### PR TITLE
add travis ci

### PR DESCRIPTION
**Pull request by little-dude**

### CODE REVIEW

**Comment by little-dude** [comment]
@t00f https://github.com/nuagenetworks/monolithe/pull/16/commits/31733080a7405717c5005ed04d745d12413842ac does not look good. The output does not have the "futurize" changes. Did this test ever pass?

**Comment by little-dude** [comment]
from https://github.com/nuagenetworks/monolithe/commit/8da73df65e028cf59e0fb14b61d0c40d9c97a264, it looks like @primalmotion ran `futurize` everywhere, including the tests, but the templates never got updated.

File: tests/base/sdk/python/tdldk/cli/commands.py [hunk 1]
```diff
@@ -34,20 +32,20 @@ def list(cls, args):
...
+        plural_classname = Utils.get_entity_name_plural(classname)
         fetcher_name = Utils.get_python_name(plural_classname)
         query_parameters = cls._get_attributes(args.query_parameters)
 
         try:
             fetcher = getattr(parent, fetcher_name)
         except:
 
-            if parent.rest_name == inspector._get_sdk_package().SDKInfo.root_object_class().rest_name:
+            if parent.rest_name == inspector._get_package().SDKInfo.root_object_class().rest_name:
```
Comment (t00f) [inline tests/base/sdk/python/tdldk/cli/commands.py]: Is that expected to change the name here ?

File: tests/base/sdk/python/tdldk/cli/commands.py [hunk 2]
```diff
@@ -69,20 +67,20 @@ def count(cls, args):
         """
         inspector = SDKInspector(args.version)
         name = Utils.get_singular_name(args.name)
-        instance = inspector.get_sdk_instance(name)
```
Comment (t00f) [inline tests/base/sdk/python/tdldk/cli/commands.py]: And here ?

**Comment by t00f** [comment]
Thanks @little-dude

**Pull request merged**